### PR TITLE
1.10.2 turtle power

### PR DESCRIPTION
**Pull request by okushama**

### CODE REVIEW

File: src/main/java/net/tropicraft/core/common/entity/underdasea/EntitySeaTurtle.java [hunk 1]
```diff
@@ -1,25 +1,580 @@
...
+			return;
+		}
+
+		if (renderSize >= 1f) {
+			setMature();
+			timeSinceLastEgg++;
+		}
+
+		if (this.timeSinceLastEgg > EGG_INTERVAL_MINIMUM && !isSeekingLand && rand.nextInt(NEST_SITE_SEARCH_ODDS) == 0
+				&& world.getWorldTime() > 13000L && world.getWorldTime() < 23000L) {
```
Comment (Corosauce) [inline src/main/java/net/tropicraft/core/common/entity/underdasea/EntitySeaTurtle.java]: This wouldnt trigger after the first night in world, suggestion: (worldTime % 24000)

**Review by Corosauce — CHANGES_REQUESTED** [review]
Nice touch with the riding how it lets you control or they take control. Spawned logs of turtles, seems performant. Aside from the world time issue all seems good.

**Review by Corosauce — APPROVED** [review]
:+1:

**Review by Cojomax99 — APPROVED** [review]
🐢

**Pull request merged**